Template parameter 0 cannot be used in an untyped template substitution. Instead of "#{}", indicate the template parameter's type with "#{any(java.lang.String)}".

## Issue

When we execute the following recipe `org.openrewrite.quarkus.spring.SpringBootToQuarkus` on a Spring Boot TODO application, we got this error:

```
~/code/quarkus/rewrite-demo/migration-tool/applications/spring-boot-todo-app on main
❯ mvn org.openrewrite.maven:rewrite-maven-plugin:dryRun \
         -Drewrite.activeRecipes=org.openrewrite.quarkus.spring.SpringBootToQuarkus \
         -Drewrite.recipeArtifactCoordinates=org.openrewrite.recipe:rewrite-spring-to-quarkus:0.5.0
...
[ERROR] Failed to execute goal org.openrewrite.maven:rewrite-maven-plugin:6.30.0:dryRun (default-cli) on project app: Execution default-cli of goal org.openrewrite.maven:rewrite-maven-plugin:6.30.0:dryRun failed: Error while visiting applications/spring-boot-todo-app/src/main/java/com/todo/app/controller/TaskController.java: java.lang.IllegalArgumentException: Template parameter 0 cannot be used in an untyped template substitution. Instead of "#{}", indicate the template parameter's type with "#{any(java.lang.String)}".                                                                                                   
[ERROR]   org.openrewrite.java.internal.template.Substitutions.substituteUntyped(Substitutions.java:183)
[ERROR]   org.openrewrite.java.internal.template.Substitutions.lambda$substitute$0(Substitutions.java:83)
[ERROR]   org.openrewrite.internal.PropertyPlaceholderHelper.parseStringValue(PropertyPlaceholderHelper.java:116)
[ERROR]   org.openrewrite.internal.PropertyPlaceholderHelper.replacePlaceholders(PropertyPlaceholderHelper.java:92)
[ERROR]   org.openrewrite.java.internal.template.Substitutions.substitute(Substitutions.java:61)
[ERROR]   org.openrewrite.java.JavaTemplate.doApply(JavaTemplate.java:123)
[ERROR]   org.openrewrite.java.JavaTemplate.apply(JavaTemplate.java:109)
[ERROR]   org.openrewrite.quarkus.spring.WebToJaxRs$WebToJaxRsVisitor.visitAnnotation(WebToJaxRs.java:290)
[ERROR]   org.openrewrite.quarkus.spring.WebToJaxRs$WebToJaxRsVisitor.visitAnnotation(WebToJaxRs.java:62)
[ERROR]   org.openrewrite.java.tree.J$Annotation.acceptJava(J.java:229)
[ERROR]   org.openrewrite.java.tree.J.accept(J.java:55)
[ERROR]   org.openrewrite.TreeVisitor.visit(TreeVisitor.java:242)
[ERROR]   org.openrewrite.TreeVisitor.visitAndCast(TreeVisitor.java:309)
[ERROR]   org.openrewrite.java.JavaVisitor.lambda$visitVariableDeclarations$22(JavaVisitor.java:906)
[ERROR]   org.openrewrite.internal.ListUtils.map(ListUtils.java:245)
[ERROR]   org.openrewrite.internal.ListUtils.map(ListUtils.java:269)
```

--- a/src/main/java/org/openrewrite/quarkus/spring/WebToJaxRs.java
+++ b/src/main/java/org/openrewrite/quarkus/spring/WebToJaxRs.java
@@ -280,7 +280,7 @@ public class WebToJaxRs extends Recipe {
                     String newAnn = "@PathParam";
                     Object[] args = new Object[0];
                     if (ann.getArguments() != null && !ann.getArguments().isEmpty()) {
-                        newAnn = "@PathParam(#{})";
+                        newAnn = "@PathParam(#{any()})";
                         args = ann.getArguments().toArray();
                     }
                     return JavaTemplate.builder(newAnn)
@@ -297,7 +297,7 @@ public class WebToJaxRs extends Recipe {
                     String newAnn = "@QueryParam";
                     Object[] args = new Object[0];
                     if (ann.getArguments() != null && !ann.getArguments().isEmpty()) {
-                        newAnn = "@QueryParam(#{})";
+                        newAnn = "@QueryParam(#{any()})";
                         args = ann.getArguments().toArray();
                     }
                     return JavaTemplate.builder(newAnn)
@@ -314,7 +314,7 @@ public class WebToJaxRs extends Recipe {
                     String newAnn = "@HeaderParam";
                     Object[] args = new Object[0];
                     if (ann.getArguments() != null && !ann.getArguments().isEmpty()) {
-                        newAnn = "@HeaderParam(#{})";
+                        newAnn = "@HeaderParam(#{any()})";
                         args = ann.getArguments().toArray();
                     }
                     return JavaTemplate.builder(newAnn)

